awslambdaric/bootstrap.py

```diff
@@ -296,8 +301,22 @@ def __init__(self, log_sink):
 
     def emit(self, record):
         msg = self.format(record)
+        self.log_sink.log(msg)
+
 
-        self.log_sink.log(msg, frame_type=getattr(record, "_frame_type", None))
+class LambdaLoggerHandlerWithFrameType(logging.Handler):
+    def __init__(self, log_sink):
+        super().__init__()
```

Comment:
Small inconsistency with the original `LambdaLoggerHandler` definition - that is using directly the base class methods:
```
logging.Handler.__init__(self)
```

Would you recommend using `super()` in both places?

Reply:
I would `super()` for both.